src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritServer.java

```diff
@@ -914,14 +915,21 @@ private void rename(String newName) {
      * @param oldName the old name of the Gerrit server
      */
     private void changeSelectedServerInJobs(String oldName) {
-        for (AbstractProject job : PluginImpl.getConfiguredJobs_(oldName)) {
-            GerritTrigger trigger = (GerritTrigger)job.getTrigger(GerritTrigger.class);
+        for (Job job : PluginImpl.getConfiguredJobs_(oldName)) {
+
+            // TODO: find a way to handle add/remove of triggers
+            if (!(job instanceof AbstractProject)) {
+                return;
```

Comment:
Does WorkflowJob have a way to add and remove triggers? I assume we'd want something similar to that? I know that the ParameterizedJobMixin does not have support for add triggers, but maybe we can just instanceOf to WorkflowJob here? 

According to the source for WorkflowJob.java, it appears that WorkflowJob has an "addTrigger" function and getTriggers function, but this isn't exposed by the Mixin API. It would feel really awkward to instanceof and then use the same code both times.... Also, it doesn't support "getTrigger" standalone only the getTriggers...

Kind of ugly either way.